src/KMonad/Keyboard/Keycode.hs

```diff
@@ -382,6 +382,8 @@ aliases = Q.mkMultiMap
   , (KeyScrollLock,     ["scrlck", "slck"])
   , (KeyScrollUp,       ["scrup", "sup"])
   , (KeyScrollDown,     ["scrdn", "sdwn", "sdn"])
+  , (KeyCopy,           ["copy"])
+  , (KeyPaste,          ["pste"])
```

Comment:
For a key named `KeyName` we automatically generate an alias of `name`, so `copy` is superfluous here. `pste`, however, seems fine.

Reply:
Thanks, I think I'll just use `paste` as the alias.